docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -108,13 +108,41 @@ The Appendix for the GBD 2017 Cause of Death manuscript is also available and Op
 Modeling non-fatal outcomes (Outline)
 ---------------------------
 
-- Figure of process
 .. image:: gbd_nonfatal_flowchart_sim_team.jpg
    :width: 600
 
-**Modeling approach** 
-
-- Modeling tools like DisMod, ST-GPR
+**DisMod** 
+
+Disease Model Meta Regression 2.1 (DisMod, sometimes DisMod MR) is a statistical modeling tool developed for the Global
+Burden of Disease study to estimate non-fatal disease burden. It is the most frequently used tool for non-fatal modeling
+in the GBD and models that are run outside of it are frequently called *custom models*. 
+
+DisMod has a user interface. The 
+`External Version
+<https://vizhub.healthdata.org/epi/>`_
+shows published and final models for GBD 2017.
+The 
+`Internal Version
+<https://internal.ihme.washington.edu/epi/>`_
+is where new and ongoing modeling occurs and is for GBD 2019 (but shows GBD 2017 best models). 
+
+There are three main components to DisMod. The first is that it
+is a meta-regression statistical model. This means that it uses point estimates with uncertainty around those estimates and covariates
+to predict disease burden for every location/year/age/sex. The second is that it is a compartmental model of disease. This
+means that DisMod is solving differential equations while fitting meta-regression estimates to enforce consistency between
+disparate measures of disease like incidence, prevalence, remission, and excess mortality. The third is that DisMod is age-integrating.
+Input data in DisMod must have an age or age range associated with those data. DisMod can account for ranges in age in the input
+data to help it produce continuous estimates of disease burden across all ages.
```

Comment:
Maybe we should bold or underline "continuous" estimates across all ages in order to emphasize this point because my understanding that this is not the case for final gbd results which are categorized by age group (assuming my understanding is accurate)... I'm only brainstorming here, but maybe this could be helpful for us to use one day for under 5 models or something that have some variation within age bins

Reply:
Yeah, I'm still a little fuzzy about what exactly is meant by "age-integrating," why this is important, and why this aspect of DisMod is described at the same level as the other two components.

Reply:
Age-integrating is a term I made up, to refer to how DisMod-MR can use data on very different age groups together.  It literally does an integral of the age-specific rate to figure out what the level should be for age a0 to a1. This is necessary when different studies report data for different age groups.  It is perhaps not too important for our simulation science work, but you can see how I define and defend this approach in Chapter 5 of _An Integrative Metaregression Framework for Descriptive Epidemiology_:
![image](https://user-images.githubusercontent.com/51236/71379124-1ca6f700-257f-11ea-97fd-25181bda1d52.png)